Add twitter embeds processor

manuq opened the pull request:
https://phabricator.endlessm.com/T23952

dnohales commented on lib/processors/twitter-embeds.js:
```diff
@@ -0,0 +1,100 @@
+
+    await Promise.all(promises.get());
+
+    return { $body, assets: assets };
```
```
{ $body, assets }
```

manuq commented:
Thanks for the review. I also removed one indentation by returning early if the tweet doesn't have pics or videos. And I renamed the two internal functions.
